cssApp.css

```diff
@@ -50,14 +53,57 @@ html, body {
     color: #555555;
 }
 
+/* backgrounds */
+
+#initial-background {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    max-height: calc(100% - 50px);
+}
+#world-map-background {
+    position: absolute;
+    max-height: 100%;
+    filter: blur(1.5px);
+}
+#headline {
+    animation: enlargeText 0.7s ease-out;
+    position: absolute;
+    top: 100px;
+    width: 50%;
+    margin: 0 auto;
+    max-width: fit-content;
```

Comment:
Be aware that this will not work on all browsers.

Reply:
Do you mean top property?
`https://caniuse.com/#search=top`